### ISSUE TITLE
Exception in some cases

### ISSUE THREAD

**Issue by ivanahepjuk** [ivanahepjuk/xmr_seed_generator#2]
When first run there is "1", the first number in hexal string is 0 - in this case its working.
When there is different first number in hexal string, its throwing an exception.

**Comment by tomichec** [comment]
When the first number is 1, the hexadecimal string has a length 65, which causes the issue in the mn_encode function.

**Comment by tomichec** [comment]
as the mn_encode is supposed to accept 64 digits

**Comment by ivanahepjuk** [comment]
You are right :O I tested this as one of the first steps, I really think i tried to pass mn_decode('1234', 'english') and it worked.. well, obviously I was wrong.
